packages/calypso-e2e/src/element-helper.ts

```diff
@@ -75,3 +75,30 @@ export async function clickNavTab( page: Page, name: string ): Promise< void > {
 	const elementHandle = await page.waitForSelector( selectors.navTab( name ) );
 	await Promise.all( [ page.waitForNavigation(), elementHandle.click() ] );
 }
+
+/**
+ * Retry any action up to three times or action passes without throwing an exception,
```

Comment:
We should add here that the function passed in must throw to trigger the retry! Some people do this pattern where the function returns a "falsey" value to trigger the retry, so we should just make sure it's clear that throwing needs to be the error case!

Reply:
Good point, text here will be updated in next revision.

Reply:
Fixed in 78d067d242bf8c2e0f9d2e842a9e2f4b84497b15.